packages/client/components/TopBarNotifications.tsx

```diff
@@ -32,6 +32,7 @@ const TopBarNotifications = ({queryRef}: Props) => {
               node {
                 id
                 status
+                type
```

Comment:
-1 Somehow this was sneaking in recently, but it should not be here since it's not used in the file directly. Instead if it's needed somewhere else, then a fragment should be spread here. @igorlesnenko can this be removed here safely?

Reply:
It could be removed, looks like some merge side effect